pkgs/development/interpreters/erlang/generic-builder.nix

```
@@ -13,6 +15,7 @@
 , perl
 , runtimeShell
 , autoconf
+, zlib
```

```suggestion
, zlib
```
Would be ideal if alphabetical order was respected.

---

We don't actually need zlib, so it's removed.